packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -299,7 +299,7 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                     display: false,
                     stacked: false,
                     ticks: {
-                        max: this.state.allMax,
+                        max: this.state.allMax > 0 ? this.state.allMax : 100,
```

Comment:
The xy-output-component was recently refactored to extract common code for the overview view. This particular line is now in the file `xy-output-component-utils.tsx`. You need to redo the change based on the recent version master branch. A rebase one just work.

Reply:
Hi Hriday, I did the refactoring for the xy-output-component. The function that this code was moved into is `getDefaultChartOptions` in `xy-output-component-utils.tsx`.